src/khoj/interface/web/config.html

```diff
@@ -1,14 +1,68 @@
-<!DOCTYPE html>
-    <head>
-        <meta charset="utf-8">
-        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🦅</text></svg>">
-        <link rel="stylesheet" href="static/assets/config.css">
-        <title>Khoj - Configure App</title>
-    </head>
-    <body>
-        <form id="config-form">
-        </form>
-        <button id="config-regenerate">regenerate</button>
-    </body>
-    <script src="static/assets/config.js"></script>
-</html>
+{% extends "base_config.html" %}
+{% block content %}
+<h2>Content Types</h2>
+<div id="content-configuration">
+    <button onclick="window.location.href='/config/content_type/pdf';">
+        Configure PDF
```

Comment:
"Configure" maybe redundant here, given we're on the Configure page of Khoj? Basing these suggestions on prior art (e.g Apple, Gmail settings)

```suggestion
        PDF
```